### ISSUE TITLE
Slider

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#141]
"Learn More" receives focus before the slider dot under the page banner despite the slider dot appearing first. Ensure focus moves left-to-right and top-to-bottom.

remove scroll jacking from the slider. This is not accessible. 

make sure inset is enabled

When keyboard-only and screen reader users activate the pagination dots in the carousel, a new slide is revealed. As users proceed to navigate, focus doesn't go to the new slide's contents right away. Ensure focus goes to the new slide after users have activated the pagination dots.

The currently active carousel pagination button is only identified visually and is not programmatically conveyed to assistive technologies. Ensure the active pagination button includes aria-current="true" so the currently selected carousel slide is properly identified.
